packages/piqilib/piqilib.0.6.15/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis:
+  "The Piqi library -- runtime support for multi-format Protobuf/JSON/XML/Piq data serialization and conversion"
+authors: "Anton Lavrik <alavrik@piqi.org>"
+maintainer: "Anton Lavrik <alavrik@piqi.org>"
+homepage: "http://piqi.org"
+bug-reports: "https://github.com/alavrik/piqi/issues"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+  [make "ocaml"]
+  [make "test"] {with-test}
+]
+install: [
+  [make "install"]
+  [make "ocaml-install"]
+]
+remove: [
```

Comment:
This and the relative `flags` Are no longer needed since opam2

Reply:
@mseri can you clarify? Do you mean the whole remove section is no longer needed? How does opam know what to remove without it?

For the flags, do you mean the `flags: light-uninstall` line?

Reply:
Yes, you can remove the whole remove and flags sections. Opam2 tracks all installed packages in its folder. In the end, you need to keep the remove and flags sections only if you are making changes to files in other packages that need to be reverted.

Reply:
@mseri this is done now